sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/TestUtils.java

```diff
@@ -31,6 +31,8 @@
  * Contains helper methods for working with AMQP messages
  */
 final class TestUtils {
+    static final String TEST_CONNECTION_STRING = "Endpoint=sb://test-event-hub.servicebus.windows.net/;SharedAccessKeyName=dummyaccount;SharedAccessKey=ctzMq410TV3wS7upTBcunJTDLEJwMAZuFPfr0mrrA08=;EntityPath=non-existent-hub;";
```

Comment:
This `SharedAccessKey` looks legit.

Reply:
Ahaha. It's definitely not. 😂 I'll make it more fake looking